packages/cli-e2e/__tests__/angular.specs.ts

```diff
@@ -130,7 +130,7 @@ describe('ui:create:angular', () => {
 
     it('should not contain console errors nor warnings', async () => {
       await page.goto(searchPageEndpoint, {
-        waitUntil: 'networkidle0',
+        waitUntil: 'networkidle2',
```

Comment:
What does motivate you to change from idle0 to idle2?

Reply:
I came across this [issue](https://github.com/puppeteer/puppeteer/issues/1908)